pkg/dceerr/error.go

```diff
@@ -0,0 +1,208 @@
+package dceerr
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"io"
+	"net/http"
+)
+
+const (
```

Comment:
A comment would nice here on what these are about (looks like this are used in API error response `Code` JSON)